Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift

```diff
@@ -64,8 +64,12 @@ extension PhotonActionSheetProtocol {
             
             let controller = ThemedNavigationController(rootViewController: settingsTableViewController)
             // On iPhone iOS13 the WKWebview crashes while presenting file picker if its not full screen. Ref #6232
-            if UIDevice.current.userInterfaceIdiom == .phone {
-                controller.modalPresentationStyle = .fullScreen
+            if #available(iOS 14.0, *){
+                // Fixed for iOS 14.0 and up
+            } else {
+                if UIDevice.current.userInterfaceIdiom == .phone {
+                    controller.modalPresentationStyle = .fullScreen
+                }
```

Comment:
nit: format. What do you think?
```suggestion
            // Fix for versions less than 14.0
            if #available(iOS 14.0, *) {} else if UIDevice.current.userInterfaceIdiom == .phone {
                controller.modalPresentationStyle = .fullScreen
            }
```